Update Japanese translation (2024-02-17)

jw-shin1abe opened the pull request:
* Base translation was created by Tilt.
* Updated by jw-shin1abe on 2024/02/17

jw-shin1abe committed:
Update Japanese translation (2024-02-17)

* Base translation was created by Tilt.
* Updated by jw-shin1abe on 2024/02/17

eumagga0x2a commented on avidemux_ja.ts:
```diff
@@ -914,12 +914,12 @@ Drop timing informations ?</source>
         <location line="+11"/>
         <source>Stat&amp;us bar</source>
-        <translation type="unfinished"></translation>
+        <translation></translation>
```
I would prefer to err on the side of caution and not use empty strings when a source string should remain untranslated. Not further delaying this PR, just as a note.

eumagga0x2a commented:
Thank you very much, my apologies for the delay.

jw-shin1abe commented:
Hi, Euma

I've just updated only two parts of tranlation in avidemux_ja.ts 
PR:  Fixed mistranslations (2024-03-16) #46 
===================================
1.  source -- Phase:
    translation OLD -- 位相:
    translation NEW -- 状態:

2.  source -- Less than a minute
    translation OLD -- 分未満
    translation NEW -- １分以下
===================================

Those my OLD translations were very stupid and make no-sense for Japanese language users. 
If you could approve my PR and commit into the current avidemux2_i18n tree, I will be appreciate.

Thanks for your support.

-- 
JW (ABE Shin-ichi ) ***@***.***>
